webapp/src/components/ProgramCalendar.tsx

```diff
@@ -65,6 +75,54 @@ const ProgramCalendar = ({
   const [dialogContents, setDialogContents] = useState<CalendarEvent>(
     defaultDialogContents
   );
+  const [activitiesCompletion, setActivitiesCompletion] =
+    React.useState<ParticipantActivityForProgram>({
+      program_id: 0,
+      participant_activities: [],
+    });
+
+  const updateActivity = (updatedActivity: CalendarEvent) => {
+    const updatedCalendarActivities = calendarActivities.map(
+      calendarActivity => {
+        if (
+          calendarActivity.curriculum_activity_id ===
+          updatedActivity.curriculum_activity_id
+        ) {
+          return updatedActivity;
+        } else {
+          return calendarActivity;
+        }
+      }
+    );
+    setCalendarActivities(updatedCalendarActivities);
+  };
+
+  const { data: participantActivitiesCompletionList, error } =
+    useApiData<ParticipantActivityForProgram>({
```

Comment:
can we do a little tweak here to make it like other API call
`  const { 
    data: participantActivitiesCompletionList, 
    error 
  } = useApiData<ParticipantActivityForProgram>({`

Reply:
Unfortunately this will cause our `yarn delint` checks to fail :( good catch though!